package.json

```diff
@@ -34,6 +34,7 @@
 		"jsdom": "^24.1.1",
 		"lint-staged": "^15.2.9",
 		"prettier": "^3.3.3",
+		"sass-embedded": "^1.79.3",
```

Comment:
question: when running npm start i am getting an error about `sass` not installed after running `npm i` when switching to the branch. i'm not seeing it in the `devDependenceies` and i just want to check if it is something i am doing wrong 😅

Reply:
Huh, interesting. The error doesn't show up on my end so not sure what's up. Did you try to see if it works in the preview? If it works there and doesn't work when you checkout the branch, maybe try deleting the node modules and do `npm ci`?